app/src/components/creatorIdentity.tsx

```diff
@@ -0,0 +1,21 @@
+import { COINBASE_VERIFIED_ACCOUNT_SCHEMA_ID } from "@/config/const";
+import { Avatar, Badge, Name, Identity, Address } from '@coinbase/onchainkit/identity';
+
+
+const CreatorIdentity = ({ address }: { address: string }) => {
+  return (
+    <Identity
+      className="bg-transparent"
+      schemaId={COINBASE_VERIFIED_ACCOUNT_SCHEMA_ID}
```

Comment:
I think schemaId is better to be set at provider like this
https://github.com/coinbase/onchainkit?tab=readme-ov-file#components

Reply:
You're right. I will move the schemaId setting to the top level.